Sources/StreamChat/Workers/Background/AttachmentUploader_Tests.swift

```diff
@@ -66,11 +66,11 @@ final class AttachmentUploader_Tests: StressTestCase {
 
             // Assert attachment is in `.pendingUpload` state.
             XCTAssertEqual(attachment.localState, .pendingUpload)
-
+            
             // Wait attachment uploading begins.
             AssertAsync.willBeEqual(
-                apiClient.uploadFile_endpoint.flatMap(AnyEndpoint.init),
-                AnyEndpoint(.uploadAttachment(with: attachmentId, type: attachment.attachmentType))
+                apiClient.uploadFile_attachment?.id,
+                attachment.asAnyModel()?.id
```

Comment:
if both are `nil` it can produce false positive. We should probably `XCTUnwrap` the `attachment.asAnyModel()?.id` value.

Reply:
Nice catch!